.github/workflows/codecov.yaml

```diff
@@ -1,6 +1,8 @@
 name: Code Coverage with codecov
 
 on:
+  workflow_dispatch:
+    branches: [main]
```

Comment:
Is this intentional? or a leftover from testing?

Reply:
removed.